script.js

```diff
@@ -0,0 +1,114 @@
+const colors = [
+    "#0fbcf9",
+    "#00d8d6",
+    "#05c46b",
+    "#ffc048",
+    "#ffdd59"
+  
+    ];
+
+  
+    let firstColor = colors[Math.floor(Math.random() * colors.length)];
+    let secondColor = colors[Math.floor(Math.random() * colors.length)];
+    
+    document.body.style.background = `linear-gradient(${firstColor}, ${secondColor})`;
+   
+    const toDoButton = document.getElementById("toDoButton");
+    const toDo = document.getElementById("toDo");
+    const toDoList = document.getElementById("todo-list");
+    const toDoInfo = document.getElementById("toDoInfo");
+    const doneList = document.getElementById("done-list");
+    const doneInfo = document.getElementById("doneInfo");
+    const finishtoDo = document.getElementById("finishtoDo");
+    
+    var toDoNum = 0;
+    var doneNum = 0;
+
+    function finish(){
+        if(toDoNum === 0){
+          finishtoDo.classList.remove("hidden");
+    
+        }
+    
+        else{
+          finishtoDo.classList.add("hidden");
+        }
+    
+       
+      }
+
+    function deleteToDo(event) {
+        toDoNum--;
```

Comment:
현재님께서 말씀하신 대로, `toDoNum`이 음수로 넘어가는 문제가 있는데 이렇게 해결해볼 수 있겠죠?
```suggestion
        toDoNum = toDoNum =< 0 ? 0 : toDoNum--
```

Reply:
감사합니다 !!!ㅠㅠ 음수로 넘어가는 거를 어떻게 할까 고민했는데 삼항 연산자로 쉽게 해결되네요! 사실 삼항 연산자를 잘 사용하지 않았는데 이번에 다시 알아갑니다 bbb